go.mod

```diff
@@ -0,0 +1,86 @@
+module github.com/konflux-ci/build-definitions
```

Comment:
Could you move the go.mod and go.sum to the remote-task-generator subdirectory?

Reply:
:+1: Yeah, that would help with EC-690, move of tash for #1039 in here.

Reply:
I might rename it to task-generator then, once this is in we can look at moving tash